components/ProfileInfo/AboutInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from "../ProfileInfo/ProfileInfo.module.css";
+import { Typography } from "@mui/material";
+
+type AboutInfoProps = {
+    name: string,
+    role: string,
+    last_active: string,
```

Comment:
for variable names and JSON attribute names, let's stick to using camel case convention within our TS/JS files
```suggestion
    lastActive: string,
```